TRANSLATORS.md

```diff
@@ -84,6 +84,13 @@ file:
 
   * Open the *.po file and update translation. Once done, as usual, check your
     translation.
+	
+### ADDING THE TRANSLATION
```

Comment:
:) And it is working now! Super cool! I can see my language. I already see
a minor language mistake. I will fix it right away.

Thank you.

2016-04-15 9:36 GMT+01:00 Robin Hahling <notifications@github.com>:

> In TRANSLATORS.md
> <https://github.com/Rolinh/dfc/pull/3#discussion_r59842959>:
>
> > @@ -84,6 +84,13 @@ file:
> >
> >    * Open the *.po file and update translation. Once done, as usual, check your
> >      translation.
> > +	
> > +### ADDING THE TRANSLATION
>
> Thanks for that!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Rolinh/dfc/pull/3/files/83dc2de21b4793d9f55fb31a632d45ebbf558f8d..e631ea38e9eb7118b7f9bd79213d57a7f9e3bc96#r59842959>
>



-- 
Sincerely,

Michiel Pater
Lead Developer
mpater@tradosoft.com

Tradosoft
Husselmansgoed 23
3863 BV Nijkerk (Gld)
The Netherlands
+316 27 47 15 86
http://www.tradosoft.com/